pkg/crd/markers/validation.go

```diff
@@ -299,12 +298,12 @@ type KubernetesDefault struct {
 // "delete"}`). Examples should be defined in pruned form, and only best-effort
 // validation will be performed. Full validation of an example requires
 // submission of the containing CRD to an apiserver.
+// +controllertools:marker:generateHelp:category="CRD validation"
 type Example struct {
 	Value interface{}
 }
 
-// +controllertools:marker:generateHelp:category="CRD processing"
-// PreserveUnknownFields stops the apiserver from pruning fields which are not specified.
+// XPreserveUnknownFields stops the apiserver from pruning fields which are not specified.
```

Comment:
Not sure if having the X prefix is wanted here / how this results later on in CRD's (if it even does)

Reply:
I think it has no impact. It's just the godoc of a marker